src/groups/bmq/bmqp/bmqp_schemalearner.cpp

```diff
@@ -45,6 +45,8 @@ struct SchemaLearner::SchemaHandle {
     // If 'false', needs new translation.
 
     SchemaHandle(SchemaIdType id);
+
+    ~SchemaHandle();
```

Comment:
Let's also add `BSLS_KEYWORD_FINAL` to the class head, since we declared a non-virtual destructor?

```
struct SchemaLearner::SchemaHandle BSLS_KEYWORD_FINAL {
```